tools/manifest/sourcefile.py

```diff
@@ -674,6 +674,32 @@ def has_testdriver(self):
             return None
         return bool(self.testdriver_nodes)
 
+    @cached_property
+    def quic_nodes(self):
+        # type: () -> List[ElementTree.Element]
+        """List of ElementTree Elements corresponding to nodes in a test that
+        specify whether it needs QUIC server."""
+        assert self.root is not None
+        return self.root.findall(".//{http://www.w3.org/1999/xhtml}meta[@name='quic']")
+
+    @cached_property
+    def quic(self):
+        # type: () -> Optional[bool]
+        """Boolean indicating whether a test requires QUIC server"""
```

Comment:
Should the difference between having quic in self.script_metadata and having quic nodes to be considered obvious to the reader? If not, some explanation might help; I have to say as *this* reader I don't immediately understand the difference and would have to go digging...

Reply:
```suggestion
        """Boolean indicating whether a test requires QUIC server

        Determined by <meta> elements (`quic_nodes()`) and "// META" comments
        (`script_metadata()`).
        """
```